### ISSUE TITLE
support for redis timeseries data

### ISSUE THREAD

**Issue by encryptblockr** [joeferner/redis-commander#547]
right now timeseries data is not supported

any plans to support timeseries data?

https://github.com/RedisTimeSeries/RedisTimeSeries

**Comment by sseide** [comment]
No, not at the moment. But feel free to add an PR. And i'm happy to answer all questions related to it.

**Comment by encryptblockr** [comment]
this should be the number request
people please thumbs up tis request
